packages/graphql-codegen-resolvers-models/src/plugin.test.ts

```diff
@@ -0,0 +1,112 @@
+import { buildSchema } from "graphql";
+import { plugin } from "../src";
+
+const testSchema = buildSchema(/* GraphQL */ `
+  schema {
+    query: Query
+  }
+
+  type Query {
+    role: [ProjectRoleDetail!]!
+  }
+
+  type ProjectRoleDetail {
+    code: String!
+    name: String!
+    email: String!
+  }
+`);
+
+describe("TypeScript Resolvers Plugin", () => {
+  it("generates models with omitted fields", async () => {
+    const config = {
+      modelIntersectionSuffix: "Template",
+      mappersConfig: {
+        ProjectRoleDetail: {
+          extend: true,
+          exclude: ["name", "email"],
+        },
+      },
+      mappers: {
+        ProjectRoleDetail: "../entities#ProjectRole",
+      },
+    };
+    const output = await plugin(testSchema, [], config, {
+      outputFile: "graphql.ts",
+    });
+
+    expect(output.content).toMatchSnapshot();
```

Comment:
Prefer inline snapshots when the output is short enough
```suggestion
    expect(output.content).toMatchInlineSnapshot();
```